src/mc_rbdyn/ForceSensor.cpp

```diff
@@ -28,21 +28,21 @@ struct ForceSensorCalibData
    * should contain 13 parameters in that order: mass (1),
    * rpy for X_f_ds (3), position for X_p_vb (3), wrench
    * offset (6).
-   * @return Default calibration if the file does not exist,
-   * throws if the file is ill-formed. */
-  ForceSensorCalibData(const std::string & filename, const Eigen::Vector3d & gravity) : ForceSensorCalibData()
+   *
+   * If the file does not exist, returns a default calibration. If the file
+   * exists but its parameters are invalid, an exception will be thrown.
+   *
+   * @return false if the file was not found, true otherwise
```

Comment:
What's the point of returning a boolean?

Reply:
The point here is that we want to allow the file not to exist (e.g the user hasn't specified explicitly a configuration file), but not to have an invalid file (that would indicate a bug in the calibration, which IMHO is not something we should silently recover from).

Reply:
With the above changes checking file existance first to generate the error message, this is no longer needed.